### PR TITLE
feat(condo): DOMA-4258 cron task added, feature flag enabled, scripts fixed, code refactored

### PR DESCRIPTION
**Pull request by akarjakin**
We have some code that 

1. Sends push notification to users, who's billing accounts have freshly imported billing receipts
2. Sends push notifications to users who have billing accounts imported and their property have freshly imported billing receipts

Before this PR those functions have been executed manually via corresponding scripts.
The goal was to run those functions automatically via cron-tasks and support ability to switch it off via feature flag (growthbook)

What's done:
1. Both functions fixed (there were some minor bugs)
2. Both functions made regular via cron tasks, supporting feature flag
3. Some code refactoring done

### CODE REVIEW

File: apps/condo/bin/lib/keystone.helpers.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const path = require('path')
+const { GraphQLApp } = require('@keystonejs/app-graphql')
+
+async function connectKeystone () {
+    const resolved = path.resolve('./index.js')
+    const { distDir, keystone, apps } = require(resolved)
+    const graphqlIndex = apps.findIndex(app => app instanceof GraphQLApp)
+    // we need only apollo
+    await keystone.prepare({ apps: [apps[graphqlIndex]], distDir, dev: true })
```
Comment (AleX83Xpert) [inline apps/condo/bin/lib/keystone.helpers.js:9]: Is variable `dev` is always should be `true`?

File: apps/condo/bin/notification/send-residents-receipt-added-notifications.js [hunk 2]
```diff
@@ -11,21 +11,9 @@
...
-    const resolved = path.resolve('./index.js')
-    const { distDir, keystone, apps } = require(resolved)
-    const graphqlIndex = apps.findIndex(app => app instanceof GraphQLApp)
-    // we need only apollo
-    await keystone.prepare({ apps: [apps[graphqlIndex]], distDir, dev: true })
-    await keystone.connect()
-
-    return keystone
-}
+const { connectKeystone } = require('../lib/keystone.helpers')
```
[the remark this answers is not in the document]
Comment (akarjakin) [reply apps/condo/bin/notification/send-residents-receipt-added-notifications.js:16]: no need

**Review by AleX83Xpert — APPROVED** [review]

File: apps/condo/domains/resident/tasks/helpers/sendResidentsNoAccountNotifications.js [hunk 3]
```diff
@@ -193,16 +196,22 @@ const sendResidentsNoAccountNotificationsForPeriod = async (period, billingConte
...
+        const redisVarName = `${REDIS_LAST_DATE_KEY}-${requestPeriod}-${billingContext.id}`
 
         /**
          * This represents min value for billingReceipt createdAt to start processing from
          * 1. First of all use resendFromDt from CLI parameter if available, else
          * 2. Use createdAt value from last success script execution stored in Redis, else
          * 3. Use thisMonthStart
          */
-        const lastDt = resendFromDt ? resendFromDt.replace(' ', 'T') : await redisClient.get(redisVarName) || thisMonthStart
+        const storedLastDt = await redisClient.get(redisVarName)
```
Comment (sitozzz) [inline apps/condo/domains/resident/tasks/helpers/sendResidentsNoAccountNotifications.js:210]: Why is there an abbreviation?
```suggestion
        const storedLastDate = await redisClient.get(redisVarName)
```

**Review by sitozzz — APPROVED** [review]

**Review by DanilovMaxim13 — APPROVED** [review]

File: apps/condo/bin/lib/keystone.helpers.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+const path = require('path')
+const { GraphQLApp } = require('@keystonejs/app-graphql')
+
+async function connectKeystone () {
```
Comment (pahaz) [inline apps/condo/bin/lib/keystone.helpers.js:4]: Use common way.

**Pull request merged**